apps/coscrad-frontend/src/components/resources/playlists/playlist-detail.full-view.presenter.tsx

```diff
@@ -1,6 +1,17 @@
 import { ICategorizableDetailQueryResult, IPlayListViewModel } from '@coscrad/api-interfaces';
-export const PlaylistDetailFullViewPresenter = (
-    _: ICategorizableDetailQueryResult<IPlayListViewModel>
-): JSX.Element => {
-    return <div>playlist</div>;
-};
+import { ResourceDetailFullViewPresenter } from '../../../utils/generic-components/presenters/detail-views';
+import { EpisodePresenter } from './episode-presenter';
+export const PlaylistDetailFullViewPresenter = ({
+    name,
+    id,
+    episodes,
+}: ICategorizableDetailQueryResult<IPlayListViewModel>): JSX.Element => (
+    <ResourceDetailFullViewPresenter name={name} id={id}>
+        <div>
+            <h3> Episodes </h3>
+            {episodes.map((episode) => (
+                <EpisodePresenter episode={episode} name={name} />
```

Comment:
Do not feed in the `name` of the playlist to the `EpisodePresenter`. It should only be concerned with the episode's name which should already be on the `episode` (view model) itself.